docs/getting-started/installing/from-pre-built-release.en.md

```diff
@@ -48,21 +48,24 @@ If the flashing process fails midway through, check the connection, restart the
 
 Two serial ports are created when `Amigo` and `Bit` are connected to a PC. Sometimes Ktool will pick the wrong and flash will fail. Manually specify the serial port to overcome this issue using `-p` argument:
 
+##### Linux
+See the correct port using `ls /dev/ttyUSB*`, in the example below we use `/dev/ttyUSB0`:
 ```bash
 ./ktool-linux -B goE -b 1500000 maixpy_amigo/kboot.kfpkg -p /dev/ttyUSB1
 ```
 
-Check por names of devices manager on Windows (e.g. COM1, COM9), or list the ports on linux
-
-```bash
-ls /dev/ttyUSB*
+##### Windows
+See the correct port at Device Manager > Ports (COM & LPT), or just try one from 1 to 12, in the example below we use `COM6`:
```

Comment:
Why try 1 to 12? I frequently get COM15, 16 ... I would remove the "brute force" suggestion.

Reply:
fine, removed 👍